package.json

```diff
@@ -100,6 +100,7 @@
     "mockdate": "3.0.5",
     "moment": "2.29.1",
     "moment-timezone": "0.5.33",
+    "nanoid": "^3.1.30",
```

Comment:
Can we use an existing library instead of adding one? I think `lodash` has a util fn and we also have `crypto-js` that might have one too

Reply:
Oh! I didn't know lodash had one. Thanks for the heads up, let me try those libraries

Reply:
We also have `app/utils/guid` 😬

Reply:
Found that as well and decided to use that util. Thanks!